docs/specification/objects.md

```diff
@@ -353,14 +355,14 @@ Field name          | Value type                               | Default | Descr
 `labels`            | Set[String]                              | Implicit| Additional labels on the ingredient for describing the type or role of the ingredient (inherited from spec)
 `material`          | [Material Run](./#material-run)          | Required| Material that is this ingredient
 `process`           | [Process Run](./#process-run)            | Required| Process that the ingredient is used in
-`notes`             | String                                   | None    | Some free-form notes about the run.
+`notes`             | String                                   | None    | Some free-form notes about the run
```

Comment:
In this case, we have inconsistencies about the capitalization of the run (v line 365)